db/migrate/20191030125101_normalize_phone_number.rb

```diff
@@ -0,0 +1,56 @@
+class NormalizePhoneNumber < ActiveRecord::Migration[5.2]
+  def up
+    Member.find_each do |m|
+      # duplicate numbers in case we need the old number
+      phone_number = normalize(m.phone_number.dup)
```

Comment:
This can be a little shorter (without the `log_invalid_number`) by using the `<<` operator (append to sequences). Also, we only want to log the old emergency phone number if the member had one:

```suggestion
      new_phone_number = normalize(m.phone_number.dup)
      new_emergency_phone_number = normalize(m.emergency_phone_number.dup)
      
      comments = m.comments || ""
      comments << "\nMember had invalid phone number #{m.phone_number}" if new_phone_number.nil?
      comments << "\nMember had invalid emergency phone number #{m.emergency_phone_number}" if new_phone_number.nil? && m.emergency_phone_number.present?
```